e2e/namespace/install/cli/promote_test.go

```diff
@@ -107,6 +107,21 @@ func TestKamelCLIPromote(t *testing.T) {
 				Expect(IntegrationPodImage(nsProd, "promote-route")()).Should(Equal(IntegrationPodImage(nsDev, "promote-route")()))
 			})
 
+			t.Run("plain integration promotion update", func(t *testing.T) {
+				// Update the configmap
+				var cmData = make(map[string]string)
+				cmData["my-configmap-key"] = "I am production, but I was updated!"
+				UpdatePlainTextConfigmap(nsProd, "my-cm", cmData)
+
+				Expect(Kamel("promote", "-n", nsDev, "promote-route", "--to", nsProd).Execute()).To(Succeed())
```

Comment:
So we don't introduce new flag e.g. `--continous` and just let promote always update the integration?  Is it safe in every case?  Shouldn't we at least show some warning message when it's going to overwrite it?

Reply:
Yeah, finally I opted for this solution. When a user is doing a `promote` probably knows what he is doing (similar when he `run` an integration that will update any previous one). Definitely, we should add some output to tell the user either the Integration was created or updated (such as when we `run` them).